src/open_inwoner/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -3926,131 +3926,136 @@ msgid ""
-"gebruikers zich nog afmelden)"
+"Enable notifications for cases (if enabled, individual users can still opt "
+"out of notifications about case updates, though not of notifications that "
+"indicate that an action concerning some case is required)"
+msgstr "Activeer notificaties die bedoeld zijn ter kennisgeving met betrekking tot aanvragen. "
+"Notificaties betreffende verplichte handelingen omtrent aanvragen kunnen niet "
+"worden uitgeschakeld. Indien ingeschakeld, kunnen gebruikers "
+"ervoor kiezen zichzelf alsnog af te melden"
 
-#: open_inwoner/configurations/models.py:378
-msgid "User notifications for expiring actions"
-msgstr "Gebruikersnotificaties voor verlopende acties"
+#: open_inwoner/configurations/models.py:380
+msgid "User notifications for messages"
+msgstr "Gebruikersnotificaties voor berichten"
 
-#: open_inwoner/configurations/models.py:381
+#: open_inwoner/configurations/models.py:383
 msgid ""
-"Notify users of expiring actions (if set, individual users can still opt out)"
-msgstr ""
-"Notificaties voor verlopende acties (indien ingesteld, kunnen individuele "
-"gebruikers zich nog afmelden)"
+"Enable notifications for messages (if enabled, individual users can still "
+"opt out; messages will only be sent if the user also opts in)"
+msgstr "Activeer notificaties voor nieuwe berichten (indien actief "
+"kunnen gebruikers ervoor kiezen zichzelf alsnog af te melden)"
 
-#: open_inwoner/configurations/models.py:385
+#: open_inwoner/configurations/models.py:388
 msgid "User notifications for expiring plans"
 msgstr "Gebruikersnotificaties voor verlopende samenwerkingen"
 
-#: open_inwoner/configurations/models.py:388
+#: open_inwoner/configurations/models.py:391
 msgid ""
-"Notify users of expiring plans (if set, individual users can still opt out)"
+"Enable notifications for plans (if enabled, individual users can still opt "
+"out)"
 msgstr ""
 "Notificaties voor verlopende samenwerkingen (indien ingesteld, kunnen "
-"individuele gebruikers zich nog afmelden)"
-
-#: open_inwoner/configurations/models.py:392
-msgid "User notifications for cases"
-msgstr "Gebruikersnotificaties voor aanvragen"
+"individuele gebruikers ervoor kiezen zichzelf alsnog af temelden)"
 
 #: open_inwoner/configurations/models.py:395
+msgid "User notifications for expiring actions"
+msgstr "Gebruikersnotificaties voor verlopende acties"
+
+#: open_inwoner/configurations/models.py:398
 msgid ""
-"Notify users of updates to cases or if an action is required (if set, "
-"individual users can still opt out)"
+"Enable notifications for expring actions concerning plans (if enabled, "
+"individual users can still opt out by disabling notifications for plans)"
 msgstr ""
-"Notificaties voor verplichte acties (indien ingesteld, kunnen individuele "
-"gebruikers zich nog afmelden)"
+"Notificaties voor verlopende acties (indien ingesteld, kunnen individuele "
+"gebruikers zich nog afmelden door noticicaties voor verlopende samenwerking "
```

Comment:
taalkundig is dit wel 'wollig', door dubbel gebruik van verlopend, dus (suggestie) misschien op het eind dan "aflopende samenwerking" gebruiken.